src/content/operations/deployment/subctl/_index.en.md

```diff
@@ -495,3 +496,32 @@ The following steps are performed:
 | `--kubeconfig` `<string>`        | Absolute path(s) to the kubeconfig file(s)
 | `--namespace` `<string>`         | Namespace in which Submariner is installed (default "submariner-operator")
 | `--yes`                          | Automatically answer yes to confirmation prompt
+
+### Running `subctl diagnose` from a Pod in cluster
+
+As of release 0.12.0, a new `subctl` image containing the subctl binary is now available in submariner repos. This can be
+used to run subctl from a Pod within the cluster. `subctl diagnose` command output can be accessed from the Pod's logs.
+A new flag, `--in-cluster` has been added to support this use case.
```

Comment:
```suggestion
`--in-cluster` has been added to support this use case.
```

Another example of wording that will go stale, should not reference the newness of a particular version.